packages/office-ui-fabric-react/src/components/Button/PrimaryButton/PrimaryButton.tsx

```diff
@@ -17,11 +16,10 @@ export class PrimaryButton extends BaseComponent<IButtonProps, {}> {
     const { theme, styles } = this.props;
 
     return (
-      <BaseButton
+      <DefaultButton
         { ...this.props }
-        variantClassName='ms-Button--primary'
```

Comment:
Well it's confusing that we are using 2 completely different sets of terminology  :) I'm trying to settle on 1.